keyboards/keebio/quefrency/keymaps/joestrong/keymap.c

```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file should have a license header.

Here's a sample:

https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15